src/raygun.rum/vendor/web-vitals-polyfills.vendor.js

```diff
@@ -1,232 +1,106 @@
-(function () {
-    // This ensures that we do not initilize Core Web Vitals for non-browser environments
-    if(typeof document === 'undefined') {
-      return;
-    }
-
-    /*
-     * Copyright 2020 Google LLC
-     *
-     * Licensed under the Apache License, Version 2.0 (the "License");
-     * you may not use this file except in compliance with the License.
-     * You may obtain a copy of the License at
-     *
-     *     https://www.apache.org/licenses/LICENSE-2.0
-     *
-     * Unless required by applicable law or agreed to in writing, software
-     * distributed under the License is distributed on an "AS IS" BASIS,
-     * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     * See the License for the specific language governing permissions and
-     * limitations under the License.
-     */
-    var firstInputEvent;
-    var firstInputDelay;
-    var firstInputTimeStamp;
-    var callbacks;
-    var listenerOpts = {
-      passive: true,
-      capture: true
-    };
-    var startTimeStamp = new Date();
-    /**
-     * Accepts a callback to be invoked once the first input delay and event
-     * are known.
-     */
-
-    var firstInputPolyfill = function firstInputPolyfill(onFirstInput) {
-      callbacks.push(onFirstInput);
+/**
+ * web-vitals v2.1.0 polyfill
+ * This comes from the google web-vital repository, polyfill script @https://github.com/GoogleChrome/web-vitals
+ */
+
+(function() {
+  // This ensures that we do not initialize Core Web Vitals for non-browser environments
```

Comment:
This is is an update we added to this script? Do we want to make it more clear that we added this block by tagging things? Not a necessity.

E.g. "Raygun: This ensures that we ..."